Maple2.File.Parser/Xml/Table/PremiumClubEffect.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using System.Xml.Serialization;
+using M2dXmlGenerator;
+
+namespace Maple2.File.Parser.Xml.Table;
+
+// ./data/xml/table/vipbenefiteffecttable.xml
+[XmlRoot("ms2")]
+public class PremiumClubEffectRoot {
+    [XmlElement] public List<PremiumClubEffect> benefit;
+}
+
+public partial class PremiumClubEffect : IFeatureLocale {
+    [XmlAttribute] public int effectID;
+    [XmlAttribute] public bool effectLevel;
```

Comment:
`effectLevel` is `short`
![image](https://user-images.githubusercontent.com/7710340/200245770-497d07d8-6e91-4e06-b5f6-d23a15c666ae.png)